plugins/GSdx/GSSetting.cpp

```diff
@@ -122,6 +122,13 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_TC_DEPTH:
 			return "Disable the support of Depth buffer in the texture cache.\n"
 				"It can help to increase speed but it will likely create various glitches.";
+		case IDC_CPU_FB_CONVERSION:
+			return "Enables 4-bit and 8-bit frame buffer conversion to be done on the CPU instead of the GPU.\n\n"
```

Comment:
IMHO, the English construction is way too complicated.

You can directly use 
`Convert 4-bit and 8-bit frame buffer on the CPU ....`